azure-pipelines/azure-templates/build-template.yml

```diff
@@ -0,0 +1,51 @@
+
+parameters:
+  - name: 'displayName'
+    type: string
+  - name: 'mavenPomFile'
+    type: string
+  - name: 'options'
+    default: ''
+    type: string
+  - name: 'targetPath'
+    type: string
+  - name: 'artifact'
+    type: string
+
+
+steps:
+- task: SonarCloudPrepare@1
+  inputs:
+    SonarCloud: 'SonarCloud'
+    organization: '1054-Daniel-Felleman-Revature-Swag-Shop'
```

Comment:
values like this that will be modified for the next iteration or a different project should be set at the pipeline level and sent here via parameters/variables.

Reply:
Removed file.